### PR TITLE
Update Properties documentation with new Optimizer property

### PR DESCRIPTION
**Pull request by abhinavmuk04**
## Description
Update Properties documentation with new Optimizer property

## Motivation and Context
Documentation update for new confidenceBasedBroadcasting property

## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/admin/properties.rst [hunk 1]
```diff
@@ -834,6 +834,16 @@ Log the stats equivalent plan and canonicalized plans used in history based opti
 Enable analysis and propagation of logical properties like distinct keys or cardinality among the nodes of
 a query plan. The optimizer may then use these properties to perform various optimizations.
 
+``optimizer.confidence-based-broadcasting``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Enable broadcasting based on the confidence of the statistics that are being used, specifically choosing to
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/admin/properties.rst:843]:
```suggestion
Enable broadcasting based on the confidence of the statistics that are being used, by
```

File: presto-docs/src/main/sphinx/admin/properties.rst [hunk 2]
```diff
@@ -834,6 +834,16 @@ Log the stats equivalent plan and canonicalized plans used in history based opti
...
+broadcast the side of a joinNode which has atleast ``HIGH`` confidence. In the case that both sides have
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/admin/properties.rst:844]:
```suggestion
broadcasting the side of a joinNode which has at least ``HIGH`` confidence. If both sides have
```

File: presto-docs/src/main/sphinx/admin/properties.rst (hunk 1 above)
Comment (abhinavmuk04) [reply presto-docs/src/main/sphinx/admin/properties.rst:843]: Thanks for the comments! Making the changes now

**Review by steveburnett — DISMISSED** [review]
LGTM! (docs)

Pull updated branch, new local docs build, looks good. Thanks!

File: presto-docs/src/main/sphinx/admin/properties.rst (hunk 1 above)
Comment (abhinavmuk04) [reply presto-docs/src/main/sphinx/admin/properties.rst]: Sorry just made a small change, based on something I changed recently. should be good to go now!

**Review by steveburnett — CHANGES_REQUESTED** [review]
Just noticed this formatting nit - the row of ^^^ should be the same character length as the text in line 837, including the `` before and after. Everything else looks good!

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

Pull updated branch, new local docs build, looks good. Thanks!

**Pull request merged**